eth_tester/validation/common.py

```diff
@@ -179,6 +179,16 @@ def validate_array(value, validator):
         raise ValidationError(error_message)
 
 
+def validate_transaction_type(value):
```

Comment:
I chose to validate here based on whether or not the value validates to `0`, `1`, or `2` integers. This is because `0x01` is the same as `0x1` and both should be accepted. Internally, we only use `0x0`, `0x1` and `0x2` for the type and this is because we pop the type out of the transaction (after we validate it here) and instead add the type that is appropriate based on the transaction parameters alone.

We do this because we don't want `type` to be a required field to send a transaction.